doc/sphinx/how_to_install.rst

```diff
@@ -16,7 +16,7 @@ Python is an open-source programming language available from https://www.python.
 
     apt-get install python python-dev
 
-NumPy (Numerical Python) adds a fast array facility and linear algebra routines to Python. It is available from https://www.numpy.org. Windows users should use the most recent NumPy installation for the Python version they have (e.g. numpy-1.0.3.1.win32-py2.4.exe for Python 2.4). Linux users are recommended to find a binary package for their distribution rather than compiling it themselves. In Debian/Ubuntu it is installed as follows (as root):
+NumPy (Numerical Python) adds a fast array facility and linear algebra routines to Python. It is available from https://numpy.org/. Windows users should use the most recent NumPy installation for the Python version they have (e.g. numpy-1.0.3.1.win32-py2.4.exe for Python 2.4). Linux users are recommended to find a binary package for their distribution rather than compiling it themselves. In Debian/Ubuntu it is installed as follows (as root):
```

Comment:
Windows users should consider using a conda installation to install Python and the most recent NumPy installation.
https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html